docs/mine/spacerace.md

```diff
@@ -112,7 +112,7 @@ Filter = "false"
 ## Accept all deals
 Filter = "true"
 
-## Only accept deals from the 4 competition dealbots
+## Only accept deals from the 4 competition dealbots (requires jq installed)
```

Comment:
```suggestion
## Only accept deals from the 4 competition dealbots (requires [jq](https://stedolan.github.io/jq/download/))
```

Reply:
This is in code-formatted text. Would you recommend the same markdown syntax or just plain link?

Reply:
Ooops, my bad. Didn't expand the code in GitHub ^^. Just saw `##` and assumed it was markdown.